pylint/pyreverse/dot_printer.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         layout: Optional[Layout] = None,
         use_automatic_namespace: Optional[bool] = None,
     ):
+        layout = layout or Layout.BOTTOM_TO_TOP
```

Comment:
Since ``DiagramWriter`` no longer passes the layout explicitly I have to set the default here in ``DotPrinter``. 
Otherwise the ``pyreverse`` output would change for either ``dot`` or ``vcg``, as they both use different layouts.